app/src/main/java/com/zello/sdk/example/app/Application.kt

```diff
@@ -1,18 +1,7 @@
 package com.zello.sdk.example.app
 
 import android.app.Application
-import com.zello.sdk.Zello
 import dagger.hilt.android.HiltAndroidApp
-import javax.inject.Inject
 
 @HiltAndroidApp
-class Application : Application() {
-
-	@Inject lateinit var zello: Zello
-
-	override fun onCreate() {
-		super.onCreate()
-		zello.start()
```

Comment:
Are we intentionally trying to discourage users from starting the SDK without the UI being shown? Or why was this invocation moved to the activity? Not a complaint, just making sure I understand.